packages/amplify-console-integration/projects-templates/create-react-app-auth-amplify/public/index.html

```diff
@@ -1,15 +1,15 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="utf-8">
-    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <meta name="theme-color" content="#000000">
+    <meta charset="utf-8" />
```

Comment:
Didnt we decide not to ship the project template as part of our repo?

Reply:
I don't recall. 
I guess Gary included it to avoid having to include `create-react-app` as a dependency and the lengthy testing project creation time .

Reply:
We should not be including anything we dont maintain as part of our tests. The code can go out of sync and our tests would not be testing the things customer is doing. So lets not ship this part of our repo.